### PR TITLE
SP-319 - Replace Timestamp with Counter

### PR DESCRIPTION
**Pull request by RaymondK-mx51**

### CODE REVIEW

File: src/Messages.js [hunk 1]
```diff
@@ -240,20 +257,19 @@ export class Message {
     }
 
     ToJson(stamp) {
-        let now = Date.now();
-        let tzoffset = new Date().getTimezoneOffset() * 60 * 1000;
-        let adjustedTime = new Date(now - tzoffset + stamp.ServerTimeDelta);
-
-        // Format date: "yyyy-MM-ddTHH:mm:ss.fff"
-        this.DateTimeStamp = adjustedTime.toISOString().slice(0,-1);
+        this.PosCounter = stamp.PosCounter ++;
```
Comment (mattgresham) [inline src/Messages.js:260]: I think the space between the variable and the increment operator needs to be removed

File: src/Messages.js [hunk 2]
```diff
@@ -240,20 +257,19 @@ export class Message {
     }
 
     ToJson(stamp) {
-        let now = Date.now();
-        let tzoffset = new Date().getTimezoneOffset() * 60 * 1000;
-        let adjustedTime = new Date(now - tzoffset + stamp.ServerTimeDelta);
-
-        // Format date: "yyyy-MM-ddTHH:mm:ss.fff"
-        this.DateTimeStamp = adjustedTime.toISOString().slice(0,-1);
```
Comment (mattgresham) [inline src/Messages.js:248]: Should the timestamp still be on the message? - https://github.com/mx51/spi-client-windows/commit/230bd13d49bd4b2eea69edf761e9e5a1c4096c4d

File: src/Messages.js (hunk 1 above)
Comment (RaymondK-mx51) [reply src/Messages.js:260]: Thanks, done.

File: src/Messages.js (hunk 2 above)
Comment (RaymondK-mx51) [reply src/Messages.js:248]: Good pick up, it works without the timestamp, but needs to be in there as part of API spec.

**Pull request merged**